includes/abstracts/abstract-wc-product.php

```diff
@@ -1871,7 +1881,8 @@ public function get_image( $size = 'woocommerce_thumbnail', $attr = array(), $pl
 	 * @return string
 	 */
 	public function get_shipping_class() {
-		if ( $class_id = $this->get_shipping_class_id() ) { // @phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.Found, WordPress.CodeAnalysis.AssignmentInCondition.Found
+		// phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.FoundInControlStructure, WordPress.CodeAnalysis.AssignmentInCondition.Found
```

Comment:
Better stop ignoring those rules, we just need to fix the `if` moving the assignment to a variable.